src/Engine/Systems/PhysicsSystem.cpp

```
@@ -404,11 +439,51 @@ void PhysicsSystem::homogenizeCoordinateSystems() {
-		std::get<PhysicsComponent::RigidBody>(m_propData[i]) = rigidBody;
-		//m_ecsRegistry->updateComponent(entityID, propagator);
-		//m_ecsRegistry->updateComponent(entityID, transform);
-		//m_ecsRegistry->updateComponent(entityID, rigidBody);
+		m_ecsRegistry->updateComponent(entityID, propagator);
+		m_ecsRegistry->updateComponent(entityID, transform);
+		m_ecsRegistry->updateComponent(entityID, rigidBody);
+	}
+}
+
+
+void PhysicsSystem::createTrajectoryPoints() {
```

Clang (14.0.6)

[warning] method 'createTrajectoryPoints' can be made static

(readability-convert-member-functions-to-static)